testapi.pm

```diff
@@ -581,6 +581,20 @@ sub click_lastmatch {
     }
 }
 
+=head2 assert_and_tclick
+
+  assert_and_tclick($mustmatch [, timeout => $timeout] [, button => $button] [, clicktime => $clicktime ] [, dclick => 1 ] [, mousehide => 1 ]);
+
+Alias for C<assert_and_click> with C<$dclick> set.
+
+=cut
+
+sub assert_and_tclick {
```

Comment:
Maybe there's some confusion here. This looks quite nice but that's not what the ticket asks for 🤓️

The AC is `mouse_tclick is covered by at least one unit test`. We don't want a new API function here, just more test coverage for `mouse_tclick`.